app/uk/gov/hmrc/helptosave/models/account/NsiAccount.scala

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.helptosave.models.account
+
+import java.time.LocalDate
+
+import play.api.libs.json.{Json, Reads}
+
+case class NsiCurrentInvestmentMonth(
+    investmentRemaining: BigDecimal,
+    investmentLimit:     BigDecimal,
+    endDate:             LocalDate
+)
+
+object NsiCurrentInvestmentMonth {
+  implicit val reads: Reads[NsiCurrentInvestmentMonth] = Json.reads[NsiCurrentInvestmentMonth]
+}
+
+case class NsiBonusTerm(termNumber: Int, endDate: LocalDate, bonusEstimate: BigDecimal, bonusPaid: BigDecimal)
```

Comment:
Does this model need to match the Term model in the stub in the NSIGetAccountBehaviour object?